src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1583,6 +1583,7 @@ private static void UpdateConnectionLimit(IEnumerable<string> urls)
 
                 try
                 {
+                    ServicePointManager.MaxServicePointIdleTime = -1;
```

Comment:
This seems like a global change that we are doing.
We can use: https://docs.microsoft.com/en-us/dotnet/api/system.net.servicepoint.maxidletime?view=netframework-4.8#System_Net_ServicePoint_MaxIdleTime

Instead, no?